src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientTest.java

```diff
@@ -47,13 +57,81 @@
 import org.apache.http.client.HttpClient;
 import org.apache.http.client.methods.HttpPost;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
 import org.mockito.Mockito;
 
 public class SnowflakeStreamingIngestClientTest {
   private static final ObjectMapper objectMapper = new ObjectMapper();
 
+  SnowflakeStreamingIngestChannelInternal channel1;
+  SnowflakeStreamingIngestChannelInternal channel2;
+  SnowflakeStreamingIngestChannelInternal channel3;
+  SnowflakeStreamingIngestChannelInternal channel4;
+
+  @Before
+  public void setup() {
+    objectMapper.setVisibility(PropertyAccessor.GETTER, JsonAutoDetect.Visibility.ANY);
+    objectMapper.setVisibility(PropertyAccessor.SETTER, JsonAutoDetect.Visibility.ANY);
+    channel1 =
+        new SnowflakeStreamingIngestChannelInternal(
+            "channel1",
+            "db",
+            "schemaName",
+            "tableName",
+            "0",
+            0L,
+            0L,
+            null,
+            "key",
+            1234L,
+            OpenChannelRequest.OnErrorOption.CONTINUE,
+            true);
+    channel2 =
+        new SnowflakeStreamingIngestChannelInternal(
+            "channel2",
+            "db",
+            "schemaName",
+            "tableName",
+            "0",
+            2L,
+            0L,
+            null,
+            "key",
+            1234L,
+            OpenChannelRequest.OnErrorOption.CONTINUE,
+            true);
+    channel3 =
+        new SnowflakeStreamingIngestChannelInternal(
+            "channel3",
+            "db",
+            "schemaName",
+            "tableName",
+            "0",
+            3L,
+            0L,
+            null,
+            "key",
+            1234L,
+            OpenChannelRequest.OnErrorOption.CONTINUE,
+            true);
+    channel4 =
+        new SnowflakeStreamingIngestChannelInternal(
+            "channel4",
+            "db",
+            "schemaName",
+            "tableName",
```

Comment:
Could you add a test to verify if there're channels belong to different table so we could verify the multiple chunks scenario?